static/app/views/onboarding/components/footer.tsx

```diff
@@ -140,16 +142,28 @@ export function Footer({projectSlug, router, newOrg}: Props) {
       source: 'targeted_onboarding_first_event_footer',
     });
 
+    const selectedProjectId = projects.find(project => project.slug === projectSlug)?.id;
+
     openChangeRouteModal({
       router,
       nextLocation: {
         ...router.location,
-        pathname: `/organizations/${organization.slug}/issues/?referrer=onboarding-first-event-footer-skip`,
+        pathname: selectedProjectId
+          ? `/organizations/${organization.slug}/issues/?project=${selectedProjectId}&referrer=onboarding-first-event-footer-skip`
+          : `/organizations/${organization.slug}/issues/?referrer=onboarding-first-event-footer-skip`,
```

Comment:
```
`/organizations/${organization.slug}/issues/?` + (selectedProjectId  ? `project=${selectedProjectId}&` : '') + `referrer=onboarding-first-event-footer-skip`;
```

Reply:
```> const res2 = `/organizations/issues/?` + (selectedProjectId  ? `project=${selectedProjectId}&` : '') + `referrer=onboarding-first-event-footer-skip`;
undefined
> res2
'/organizations/issues/?project=234&referrer=onboarding-first-event-footer-skip'
>
```
seems working

Reply:
yes, that could also work... I don't have a strong opinion but just because I just pushed...mind if I update that in a follow-up?

Reply:
yeah, it's a small one, not blocking PR, just a suggestion.

Reply:
I had to update a test so I applied your suggestion 😉